ztp/policygenerator/resources/cluster-crs.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   clusterDeploymentRef:
     name: siteconfig.Spec.Clusters.ClusterName
   imageSetRef:
-    name: siteconfig.Spec.ClusterImageSetNameRef
```

Comment:
This file is absolute the right place to change that is her https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/siteconfig-generator/siteConfig/clusterCRs.go#L25

Reply:
Done.